podman_compose.py

```diff
@@ -89,6 +89,8 @@ def parse_short_mount(mount_str, basedir):
         mount_type = "bind"
         # TODO: should we use os.path.realpath(basedir)?
         mount_src = os.path.join(basedir, os.path.expanduser(mount_src))
+        if not os.path.exists(mount_src):
```

Comment:
use `try/catch` instead a conditional prevent a unexpected issue

Reply:
Good idea! Fixed it.